uniconnect-epfl/uniconnect
# feat: navigation refinement with a `Header` and a `Footer`

# What I did
Added header and footer components to refine the navigation flow. Now, the main tabs can be traversed and different screens show up. Header component also contains icons that reference to Profile and Settings screens (that are empty for now). All in all, the navigation flow is now consistent with the flow defined in Figma.

Little detail: also changed the color of status bar to dark, since the background is light.

# How I did it
- Defined two new components: `Header` and `Footer`.
- Changed `HomeTabNavigator` and `RegistrationStackNavigator` to contain the needed navigation logic and accomodate newly created header and footer.
- Imported a `StatusBar` component to be able to control status bar color.


# How to verify it
No additional changes needed. Open the app, login and check the navigation flow yourself.

# Demo
![photo_2024-04-12_16-53-45](https://github.com/uniconnect-epfl/uniconnect/assets/79483254/189db813-88cb-4c3f-8380-ddbca3426832)
